docs/testing-with-ryujit.md

```diff
@@ -103,7 +97,52 @@ are specified without using the `.dll` file name extension.
         Value: assembly_one;assembly_two;assembly_three
 
   **Note** This is a semicolon-delimited list of simple assembly names.
+
+Disabling the use of all NGEN images
+====================================
+
+To prevent any NGEN native image from being used, and force all code to be compiled with the JIT compiler, you can use the ZapDisable configuration variable, as follows. You might choose to do this as an experiment, to see if any NGEN native image contains generated code that is inducing a bug in your application. Generally, if an NGEN native image does have a problem, and the identity of that native image can be determined, using one of the `DisableNativeImageLoadList` mechanisms described above is preferable.
+
+**Note** This setting applies to both the 32-bit and 64-bit JIT. Thus, setting this globally will affect all 32-bit .NET applications as well. This is particularly true for **Method 2: environment variable**.
+
+**Note 2** NGEN provides significant performance improvements to .NET applications. Disabling the use of NGEN will cause the perfomance of .NET applications to be significantly worse.
```

Comment:
"Disabling the use of NGEN may result in significantly slower application startup times"